recipes/glbinding/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,29 @@
+#include <glbinding/glbinding.h>
+
+#define GLFW_INCLUDE_NONE
+#include <GLFW/glfw3.h>
+
+int main() {
+#if defined(__APPLE__) && defined(__MACH__)
+    glfwInitHint(GLFW_COCOA_MENUBAR, GLFW_FALSE);
+#endif
+    if (!glfwInit()) return 1;
```

Comment:
```suggestion
    if (!glfwInit()) {
        printf("glfwInit() failed. Ignoring because this is not testing glfw.\n");
        return 0;
    }
```